Deploy/metafilemanager.cpp

```diff
@@ -254,12 +254,12 @@ void MetaFileManager::createRunMetaFiles(const QHash<QString, DeployCore::QtModu
     for (auto i = DeployCore::_config->targets().cbegin(); i != DeployCore::_config->targets().cend(); ++i) {
 
         if (!createRunScript(i.key())) {
-            QuasarAppUtils::Params::log("run script not created!",
-                                               QuasarAppUtils::Error);
+            QuasarAppUtils::Params::log("Fail to created the run script!",
+                                         QuasarAppUtils::Error);
```

Comment:
```suggestion
            QuasarAppUtils::Params::log("Failed to create a run script!",
                                         QuasarAppUtils::Error);
```

Reply:
I think it would be better if this can be rewritten as "Failed to create the run script for target <foo>". Otherwise, using "a" is better than using "the".

Reply:
@pzhlkj6612 
the = definite article
a/an = indefinite article

in this case we cannot know then **what the script** or **script of what**. it is just some script of some target. so i think that 'a' is better for this case.